### ISSUE TITLE
How to use in Windows by using Visual Studio 

### ISSUE THREAD

**Issue by umutozturk95** [mfontanini/cppkafka#101]
Hi,
Do you have a chance to explain step by step how to compile this api in VS for C++?
I got a lot of errors while compling this api and did not find enough resources . Do you have resources to provide?I tried  Cmake to compile this api,but  compile steps are not enough clear.

All the best,

**Comment by umutozturk95** [comment]
Yes , cppkafka project is  compiled successfully  VS 2015 ,but it does not run successfully. 
**unable to start program  ...\cppkafka .lib**  is occured in console.
Do you know how to solve this error?
Thank you.

**Comment by mfontanini** [comment]
cppkafka is not an executable, it's a library. You can import it into a project and use it but you can't just "run it".

**Comment by jlohiya** [comment]
@umutozturk95 were you able to link with your project?

Please help me if it got linked and working successfully, i need to integrate with my c++ application.

Thanks in advance :)

**Comment by cfinkenstadt** [comment]
@close

**Comment by latermarch** [comment]
well,I can't compile rihgtly,I meet the problems following in the pic
![image](https://user-images.githubusercontent.com/9695216/60420192-b073b800-9c19-11e9-91e5-98af108f0135.png)

**Comment by latermarch** [comment]
ok，I see the answer in other issue. @close

**Comment by nitinpatharia** [comment]
@umutozturk95  hi i am trying to build the  code but  getting lot of error .
can you please tell   steps  as how  you have  build  this  lib   steps  by  steps ,   and the what  setting  you  have used to  build the  lib .
appreciate your  help  on this .
Thanks
